docs/skopeo-copy.1.md

```diff
@@ -164,6 +164,22 @@ Bearer token for accessing the destination registry.
 
 The number of times to retry. Retry wait time will be exponentially increased based on the number of failed attempts.
 
+**--src-username**
+
+User name to access the source registry
```

Comment:
In all the man page entries: It seems that the existing convention is to terminate each description with a full stop.

Reply:
```suggestion
User name to access the source registry
```
```suggestion
The username to access the source registry.
```